site/app/Resources/views/project/settings.html.twig

```diff
@@ -170,46 +161,11 @@
       </div>
     {{ form_end(form) }}
   </div>
+  {# Passing classification data to classification.js #}
+  <div class="hidden" id="classification-data" data-classification="{{ classificationHierarchy|json_encode }}"></div>
```

Comment:
The classification hierarchy is static information and generated during (Ansible) build time (it's generated from `site/app/Resources/classifications.yml`. If you need this information in some way in JS/in a JS file, then there are two better options:
- Let Ansible create a JS/JSON file of this classifications.yml file during build time.
- Let Symfony create a JSON file from it with a separate task. (FOSJsRoutingBundle does that, as shown here: https://symfony.com/doc/master/bundles/FOSJsRoutingBundle/usage.html)

I'd prefer the second option.